libraries/consul_service.rb

```diff
@@ -55,6 +55,7 @@ def default_environment
         {
           'GOMAXPROCS' => [node['cpu']['total'], 2].max.to_s,
           'PATH' => '/usr/local/bin:/usr/bin:/bin',
+          'SHELL' => node['consul']['service_shell'],
```

Comment:
Unless @johnbellone disagrees, I'll just use `node['consul']['service_shell']` with a condition. If others, like us, want to specify a custom shell in the systemd file so the `consul` user can run script checks, they can override the template. I'll look at doing this later this week.